python/adelphi/tests/integration/test_cql.py

```diff
@@ -0,0 +1,146 @@
+import glob
```

Comment:
Another major change.  We used to have individual test classes (each contained in their own file) for a specific test case grouped within a module representing the export type we wanted to test.  The connective tissue between those tests and unittest wasn't completely intuitive; it wasn't clear which test function was actually invoked in a given case.

This PR changes all that by moving all tests for a specific export type into a single file.  Each test case now has a descriptive method name (by itself a massive improvement) and the module hierarchy was simplified.  The odd inheritance + double mixin strategy we were using has also been removed in favor of a much simpler inheritance model.

Even a cursory walk-through should make it clear how this approach _vastly_ simplifies https://github.com/datastax/adelphi/issues/110.